Ginger/Ginger/Actions/ActScriptEditPage.xaml.cs

```diff
@@ -120,10 +121,67 @@ private void ScriptInterpreterComboBox_SelectionChanged(object sender, Selection
             if (ScriptInterpreterComboBox.SelectedValue.ToString() == "Other")
             {
                 InterpreterPanel.Visibility = Visibility.Visible;
+                InterpreterPanelLabel.Visibility = Visibility.Visible;
+                ScriptNameComboBox.Items.Clear();
+                if (!Directory.Exists(SHFilesPath))
+                    Directory.CreateDirectory(SHFilesPath);
+                string[] fileEntries = Directory.EnumerateFiles(SHFilesPath, "*.*", SearchOption.AllDirectories)
+                .Where(s => s.ToLower().EndsWith(".vbs") || s.ToLower().EndsWith(".js") || s.ToLower().EndsWith(".pl") || s.ToLower().EndsWith(".bat") || s.ToLower().EndsWith(".cmd")).ToArray();
+
+                foreach (string file in fileEntries)
+                {
+                    string s = file.Replace(SHFilesPath, "");
```

Comment:
Directly bind the list of items to the combo box instead of iterating and adding one by one